src/core/control/Control.cpp

```diff
@@ -1602,6 +1634,38 @@ void Control::setViewPresentationMode(bool enabled) {
 
 void Control::setPairsOffset(int numOffset) {
     settings->setPairsOffset(numOffset);
+
+    ActionType action{};
+
+    switch (numOffset) {
+        case 0:
+            action = ACTION_SET_PAIRS_OFFSET_0;
+            break;
+        case 1:
+            action = ACTION_SET_PAIRS_OFFSET_1;
+            break;
+        case 2:
+            action = ACTION_SET_PAIRS_OFFSET_2;
+            break;
+        case 3:
+            action = ACTION_SET_PAIRS_OFFSET_3;
+            break;
+        case 4:
+            action = ACTION_SET_PAIRS_OFFSET_4;
+            break;
+        case 5:
+            action = ACTION_SET_PAIRS_OFFSET_5;
+            break;
+        case 6:
+            action = ACTION_SET_PAIRS_OFFSET_6;
+            break;
+        case 7:
+            action = ACTION_SET_PAIRS_OFFSET_7;
+            break;
+        default:
+            action = ACTION_SET_PAIRS_OFFSET;
+    }
+    
```

Comment:
What is this code segment used for? The variable `action` looks unused.

Reply:
I mistook it for an `action` in `fireActionSelected()` which is fired in `setViewColumns()`. I will remove it.